### PR TITLE
Bako branch

### PR DESCRIPTION
**Pull request by KimboGilda**
@ratnanil please have a look (review) on what I added and changed. Since we are eligible for a merge to the master we can further proceed

### CODE REVIEW

File: 02_Data_Management/Layers_and_Table_Views.Rmd [hunk 1]
```diff
@@ -1,4 +1,54 @@
-## Select by Location
+## Layers and Table Views toolset
```
Comment (ratnanil) [inline 02_Data_Management/Layers_and_Table_Views.Rmd:1]: This suggestion is a consequence of arranging everything by toolbox and toolset. From this point of view, adding "Select by location / atttribute" to "Data Management --> Layers and Table Views" is reasonalbe. However: We are not only describing Geoprocessing tools in this book, but also GUI aspects (Importing data, making maps etc). From this point of view, I think it would make more sense to keep the chapter "select" with two subchapters "select by location" and "select by attribute":

File: 02_Data_Management/Layers_and_Table_Views.Rmd [hunk 2]
```diff
@@ -130,78 +175,3 @@ data("badeplaetze_zh")
...
-swimmSpots_richt <- badeplaetze_zh[richterswil, ,op = st_within]
-```
-
-```{r,echo=F}
-ggplot() + geom_sf(data = gemeinde_zh) + geom_sf(data = badeplaetze_zh,color = "blue") + geom_sf(data = richterswil, fill = "white") + geom_sf(data = swimmSpots_richt, color = "red") + ggtitle("Swimming spots in the Canton of Zurich") + theme_minimal() + coord_sf(datum = 2056)
-```
-
-
-
-### Completely_within
```
Comment (ratnanil) [inline 02_Data_Management/Layers_and_Table_Views.Rmd:151]: if you removed these subchapters, where are they now?

File: 02_Data_Management/Projections_and_Transformations.Rmd [hunk 3]
```diff
@@ -150,13 +150,6 @@ plot(publicTransport_CH,las=1,
 
 
 
-### Rescale
```
Comment (ratnanil) [inline 02_Data_Management/Projections_and_Transformations.Rmd:153]: Where is this subchapter now?

File: 03_Data_Management/Sampling_toolset.Rmd [hunk 4]
```diff
@@ -1,36 +0,0 @@
-
```
Comment (ratnanil) [inline 03_Data_Management/Sampling_toolset.Rmd:1]: why did you delete this file?

File: 05_3D_Analyst/3D_Analyst.Rmd [hunk 5]
```diff
@@ -0,0 +1,5 @@
+# 3D Analyst Toolbox
```
Comment (ratnanil) [inline 05_3D_Analyst/3D_Analyst.Rmd:1]: I would suggest naming the file describing the toolbox "index.Rmd" (e.g. 05_3D_Analyst/index.Rmd)

Comment (ratnanil) [reply 05_3D_Analyst/3D_Analyst.Rmd:1]: Careful! this does not work, since all files are moved to the top level directory and there can be only one "index.html". So let's leave the name "3D_Analyst.Rmd" etc.

File: 02_Data_Management/Layers_and_Table_Views.Rmd (hunk 2 above)
Comment (KimboGilda) [reply 02_Data_Management/Layers_and_Table_Views.Rmd:151]: @ratnanil  based on my approach, these are now under the umbrella of Layers and Table Views -> Select by Location -> Completely within (as #### subchapter)
<

File: 03_Data_Management/Sampling_toolset.Rmd (hunk 4 above)
Comment (KimboGilda) [reply 03_Data_Management/Sampling_toolset.Rmd:1]: @ratnanil  I did not delete it. I renamed is as to Sampling.Rmd. I did that to keep some consistency. Although all Rmd file represent a toolset, only some of them included the _toolset in their naming. It's just a proposal. We can discuss it

File: 02_Data_Management/Projections_and_Transformations.Rmd (hunk 3 above)
Comment (KimboGilda) [reply 02_Data_Management/Projections_and_Transformations.Rmd:153]: @ratnanil probably I removed it by mistake. I will add it again
